x-pack/plugins/ml/public/application/components/job_selector/job_selector_table/job_selector_table.js

```diff
@@ -265,17 +264,20 @@ export function JobSelectorTable({
     <Fragment>
       <MlNodeAvailableWarningShared nodeAvailableCallback={setMlNodesAvailable} />
       {jobs.length === 0 && (
-        <EuiCallOut
+        <EuiEmptyPrompt
```

Comment:
I can't work out where this is shown! In the Single Metric Viewer or Anomaly Explorer pages inside ML, I get an empty state view in the ML pages when I have no jobs and the job selector isn't available. 

<img width="1488" alt="Screenshot 2024-07-26 at 11 50 13" src="https://github.com/user-attachments/assets/ed7d6f61-81ab-40f7-b702-bd434f0f8b73">

Reply:
Resolving comment - as discussed offline this won't occur in normal operation but the code is now there in case we ever do show the job selector in the ML pages when there are no jobs.